server/views/macros/conductReportDetails.njk

```diff
@@ -0,0 +1,81 @@
+{% macro conductReportDetails(conductReportData) %}
+    <div class='govuk-!-margin-left-8 govuk-!-margin-right-8'>
+        <div class='adjudication-pdf-section-prisoner govuk-!-width-full'>
+            <div class='adjudication-pdf-section-prisoner__item'>
+                <p class='govuk-!-margin-bottom-0 govuk-!-font-weight-bold'>Name of establishment</p>
+                <p class='govuk-!-margin-bottom-0'
+                   data-qa='adjudication-history-for-current-sentence-name-of-establishment'>
+                    {{ conductReportData.establishmentName }}
+                </p>
+            </div>
+            <div class='adjudication-pdf-section-prisoner__item'>
+                <p class='govuk-!-margin-bottom-0 govuk-!-font-weight-bold'>Prison number</p>
+                <p class='govuk-!-margin-bottom-0'
+                   data-qa='adjudication-history-for-current-sentence-prisoner-number'>
+                    {{ conductReportData.prisonerNumber }}
+                </p>
+            </div>
+            <div class='adjudication-pdf-section-prisoner__item'>
+                <p class='govuk-!-margin-bottom-0 govuk-!-font-weight-bold'>Prisoner’s full name</p>
+                <p class='govuk-!-margin-bottom-0'
+                   data-qa='adjudication-history-for-current-sentence-prisoner-full-name'>
+                    {{ conductReportData.prisonerDisplayName }}
+                </p>
+            </div>
+        </div>
+
+        <div class='adjudication-pdf-section-signature-details'>
+            <div class='adjudication-pdf-section-signature-details__item'>
+                <p class='govuk-!-margin-bottom-1 govuk-!-margin-top-3'>Key worker</p>
```

Comment:
For now I'm not pre-populating this and it will need to be written by the officer